huaweicloud/services/apig/resource_huaweicloud_apig_instance.go

```diff
@@ -44,100 +61,166 @@ func ResourceApigInstanceV2() *schema.Resource {
+				Description:  `The egress bandwidth size of the dedicated instance.`,
 			},
 			"eip_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: `The EIP ID associated with the dedicated instance.`,
+			},
+			"ipv6_enable": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: `Whether public access with an IPv6 address is supported.`,
+			},
+			"loadbalancer_provider": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(ProviderTypeLvs), string(ProviderTypeElb),
+				}, false),
+				Description: `Whether bandwith access supports IPv6.`,
 			},
 			"maintain_begin": {
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
-				ValidateFunc: validation.StringMatch(regexp.MustCompile("^(02|06|10|14|18|22):00:00$"),
+				ValidateFunc: validation.StringMatch(regexp.MustCompile(`^(02|06|10|14|18|22):00:00$`),
 					"The start-time format of maintenance window is not 'xx:00:00' or "+
 						"the hour is not 02, 06, 10, 14, 18 or 22."),
+				Description: `The start time of the maintenance time window.`,
+			},
+			// Attributes
+			"maintain_end": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `End time of the maintenance time window, 4-hour difference between the start time and end time.`,
 			},
 			"ingress_address": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The ingress eip address.`,
+			},
+			"vpc_ingress_address": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The ingress private ip address of vpc.`,
 			},
 			"egress_address": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The egress (nat) public ip address.`,
 			},
-			"create_time": {
-				Type:     schema.TypeString,
-				Computed: true,
+			"supported_features": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Description: `The supported features of the dedicated instance.`,
 			},
-			"maintain_end": {
-				Type:     schema.TypeString,
-				Computed: true,
+			"created_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Time when the dedicated instance is created, in RFC-3339 format.`,
 			},
 			"status": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Status of the dedicated instance.`,
 			},
-			"supported_features": {
-				Type:     schema.TypeList,
-				Computed: true,
-				Elem:     &schema.Schema{Type: schema.TypeString},
+			// Deprecated arguments
+			"available_zones": {
```

Comment:
add deprecated: "XXX" ?

Reply:
the Deprecated label has been added to the parameter 'create_time', according to the discussion, arguments are not necessary.